src/MPS.jl

```diff
@@ -495,6 +495,7 @@ function canonize!(ψ::AbstractMPO)
 
         # extract the singular values and contract them with the next tensor
         Λᵢ = pop!(ψ, tensors(ψ; between=(Site(i), Site(i + 1))))
+        normalize && (Λᵢ ./= norm(Λᵢ))
```

Comment:
I'm not sure why you need to normalize on canonization? Like it's completely doable, but I would prefer them to be separate because semantically are different things and there is no performance overhead when doing this:

```julia
canonize!(ψ)
normalize!(ψ)
```

Reply:
Because it's the way to normalize a state when you have an `mps` in the `Canonical` `Form`. This makes much more sense than what we have now, which involves converting the `Canonical` into `MixedCanonical` and normalizing a bond.

Reply:
Yeah, my point is to have canonization and normalization separate, and have a `normalize!` method dispatched on `Canonical` that normalizes each $\Lambda$ independently.

So... exactly taking this line and moving it to `normalize!`.